message ProposalAck {
  required uint64 zxid = 1;
}